build/images/ovs/apply-patches.sh

```diff
@@ -88,6 +88,12 @@ if version_lt "$OVS_VERSION" "2.14.1" ; then
         git apply
 fi
 
+# This patch is necessary to ensure that ovs-monitor-ipsec generates a correct IPsec configuration
+# for strongSwan when using IPv6.
+# TODO: Switch to official patch once it is merged upstream.
+curl https://github.com/antoninbas/ovs/commit/126ec7700080427bf109604d9a49bcca2c74f77d.patch | \
```

Comment:
should we also check the OVS version here for this patch?

Reply:
one question which is not related with this PR, I am wondering which one is the correct `IPsec` or `IPSec`? I saw both in our documents or codes.

Reply:
* I think this patch can be applied to all OVS versions currently supported by this script
* I believe the most "correct" one is `IPsec`, and we should try to standardize on this one

Reply:
thanks, I will check and update `IPSec` to `IPsec` in a separate PR. I am thinking maybe we can standardize some abbrs words and common review rules and document them in our contributing.md,  any suggestion? below are two main things I noticed:
1. Make sure all K8s related words start with Upper case.
2. some abbrs. like `IPsec` `Antrea` etc.

Reply:
Sounds good to me